_i18n/de/resources/moneropedia/account.md

```diff
@@ -68,14 +68,14 @@ Background refresh thread started
 
 ```
 
-As the example above indicates, it is incredibly important to store these words in safe locations. If you are concerned about the risk of critical loss at your home, for instance, you may want to store a second copy of your seed with your attorney or in a safety deposit box. It is also recommended that it is stored in a way that does not make it obvious that it is your seed, so writing it into a letter or as part of other notes is advisable.
+Wie das obige Beispiel zeigt, ist es unglaublich wichtig, diese Wörter an sicheren Orten aufzubewahren. Solltest du etwa über deren Verlust bei dir Zuhause besorgt sein, könntest du eine Kopie deines Seeds bei einem Bevollmächtigten oder in einem Tresor- oder Bankschließfach lagern. Es wird außerdem empfohlen, ihn so aufzubewahren, dass es nicht direkt ersichtlich ist, dass es sich um deinen Seed handelt; das Einbinden der Wörter in einen Brief oder Notizen ist empfehlenswert.
 
-### Practicing Good Security
+### Gute Sicherheit betreiben
 
-Over and above backing up your @mnemonic-seed so that you have access to your account in the event of critical data loss, it is also important to practice good security. Use a secure password when creating a local Monero account (not used on [MyMonero](https://mymonero.com) or other web-based account systems).
+Über das Sichern deines @mnemonischen-Seeds (wodurch dein Zugriff auf dein Konto im Falle eines totalen Datenverlusts gewährleistet ist) hinaus ist es ebenso wichtig, eine gute Sicherheit zu betreiben. Nutze ein sicheres Passwort, wenn du ein lokales Monero-Konto erstellst (eines, das du nicht auf [MyMonero](https://mymonero.com) oder anderen webbasierten Kontosystemen verwendest). 
 
-Don't ever give your Monero account password to anyone, as this can be used to access the Monero on your computer without knowing your @mnemonic-seed. Similarly, make sure you have running and up-to-date antivirus, especially on Windows computers. Finally, be careful when clicking links in emails or on unknown and untrusted websites, as malware installed on your computer can sit and wait for you to access your Monero account before taking the funds from it.
+Gebe niemals das Passwort deines Monero-Kontos an jemanden weiter: Dies kann dazu verwendet werden, auf die Moneros auf deinem Computer zuzugreifen, ohne deinen @mnemonischen-Seed kennen zu müssen. Stelle gleichermaßen sicher, dass du ein aktives und aktualisiertes Antivirenprogramm hast, insbesondere auf Windows-Computern. Sei schlussendlich vorsichtig, wenn du auf Links in E-Mails oder auf unbekannten und nicht vertraulichen Webseiten klickst; auf deinem Computer installierte Malware kann auf dem Rechner verweilen und auf deinen Zugriff auf dein Monero-Konto warten, bevor es Gelder davon entwendet.
```

Comment:
I translated the English version as it is; I didn't want to change contents by myself. Please open an issue to point this out for the EN-version so it can be changed accordingly.